examples/enable-operator-and-auto-instrumentation/spring-petclinic/02_admin-server-deployment.yaml

```diff
@@ -0,0 +1,44 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  annotations:
+    kompose.cmd: kompose --file spring-petclinic/spring-petclinic-microservices/docker-compose.yml  convert
```

Comment:
Is it reasonable to either include a makefile that goes through a (containerized) kompose process to generate these or to "de-kompose" them so they are standalone without a missing piece? This could even just include removing the annotations and merging the manifests into per-service files.

Reply:
We could use a makefile action to generate resources like this. I declined doing so at this time for a couple of reason I could use some input on. We can easily move this demo code anywhere, I will meet with stake holder more about this later this week.

- Usually we just have to update image versions for demos like this. With the spring-clinic demo, we would likely just pull the latest image.
- We need to decide where demo projects live for all auto-instrumentations libraries, this is a great time to start to figure that out. Upstream projects like https://github.com/open-telemetry/opentelemetry-helm-charts/tree/main/charts/opentelemetry-demo create a whole chart for deploying demo environments. I would like us to create our own Helm chart for demo applications in the future, however, I would need to prioritize it and evaluate the acceptance criteria with the maintainer teams.

Reply:
Longer term strategies sound good, but for these changes specifically introducing generated files without including the generator creates a missing step 0/1 for maintenance. This is especially a concern imo for customer-centric example content. If the aim is to avoid the effort to do so, merging/organizing the content in a more idiomatic way removes the requirement and can make the artifacts more intuitive.

Reply:
Got it. Working on getting low maintenance solution for this now.